crates/unftp-auth-jsonfile/key-generator.Dockerfile

```diff
@@ -0,0 +1,6 @@
+FROM alpine:latest
+
+RUN apk add bash openssl nettle-utils jq pwgen
+COPY files/run.sh /
```

Comment:
Perhaps just rename run.sh to something more descriptive like `generate-keys.sh`